public-ui/src/app/data-browser/views/ehr-view/ehr-view-react.component.tsx

```diff
@@ -270,381 +266,402 @@ interface State {
-                                    totalParticipants={totalParticipants}
-                                    selectedConcept={selectedConcept} synonymString={concept.conceptSynonyms.join(', ')}/>;
-                                })}
-                        </div>
-                        }
-                       </React.Fragment>
-                     </div>
-                     </section>
-                </div>
-                    {(concepts && numPages && numPages > 1) &&
-                     <ReactPaginate
-                              previousLabel={'Previous'}
-                              nextLabel={'Next'}
-                              breakLabel={'...'}
-                              breakClassName={'break-me'}
-                              activeClassName={'active'}
-                              pageCount={numPages}
-                              marginPagesDisplayed={2}
-                              pageRangeDisplayed={5}
-                              onPageChange={this.handlePageClick}
-                              containerClassName={'pagination'}
-                            /> }
-            </div>
-           </div>
-        </div>
+                    </div>
+                }
+                {(!loading && concepts.length === 0 && searchWord) &&
+                    <div>
+                        <h5 className='secondary-display'> No results in this domain that match your search.</h5>
+                        <NoResultSearchComponent domainMatch={this.changeResults}
+                            searchValue={searchWord}
+                            measurementTestFilter={noMatchFilter}
+                            measurementOrderFilter={noMatchFilter} />
+                    </div>}
+                {showStatement && <PopUpReactComponent helpText='EhrViewPopUp' onClose={() => this.setState({ showStatement: false })} />}
+            </React.Fragment>;
         }
-        {(!loading && concepts.length === 0 && searchWord) &&
-        <div>
-            <h5 className='secondary-display'> No results in this domain that match your search.</h5>
-            <NoResultSearchComponent domainMatch={this.changeResults} searchValue={searchWord} measurementTestFilter={noMatchFilter}
-             measurementOrderFilter={noMatchFilter} />
-        </div>}
-        {showStatement && <PopUpReactComponent helpText='EhrViewPopUp' onClose={() => this.setState({showStatement: false})} />}
-    </React.Fragment>;
-   }
-}
+    }
+);
 
 @Component({
     // tslint:disable-next-line: component-selector
     selector: 'react-ehr-view',
     template: `<span #root></span>`
 })
 
-export class EhrViewWrapperComponent extends BaseReactWrapper {
-    @Input() domainId: string;
-    @Input() searchTerm: string;
-
-    constructor() {
-        super(EhrViewReactComponent, ['domainId', 'searchTerm']);
-    }
-}
+export class EhrViewWrapperComponent extends BaseReactWrapper { }
```

Comment:
We should either leave the empty constructor here or remove the Angular component completely.  I don't think it will work like this.